### PR TITLE
612 - Fixed web help examples description typo

### PR DESCRIPTION
**Pull request by namanpatel1112**
Fixed typo in `imperative/packages/cmd/src/CommandPreparer.ts` at line 440.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Fix web help examples descroption typo at line 440 in `packages/cmd/src/CommandPreparer.ts`. [#612](https://github.com/zowe/imperative/issues/612)
```
Comment (zFernand0) [inline CHANGELOG.md:7]: `descroption` -> `description`

**Review by t1m0thyj — APPROVED** [review]
Thanks for fixing this, I've noticed this typo before in Zowe CLI and it has bugged me 😁

**Comment by t1m0thyj** [comment]
Looks like some test snapshots need to be updated.

Jest will update outdated snapshots when the `-u` flag is passed. To do this for unit tests, you can run `npm run test:unit -- -u`.

For integration tests, you can do the same: `npm run test:integration -- -u`. But since integration tests often take a while to run, it may be faster to update snapshots for the individual test files that have failing snapshots (e.g., `npx jest Cmd.cli.nested.test.ts -u`).

**Review by t1m0thyj — APPROVED** [review]
Thanks for updating the snapshots!

**Review by awharn — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
`npm update`? Nice!
Also, thanks for fixing!

**Comment by namanpatel1112** [comment]
@zFernand0 Snapshots were updated using the commands Timothy mentioned:
- npm run test:unit -- -u
- npm run test:integration -- -u

**Pull request merged**